Src/Innovative.Blazor.Components/Components/SidePanel/SidePanelComponent.razor.css

```
@@ -33,16 +33,16 @@
 
 .sidepanel-icon-button,
 .sidepanel-actions ::deep .sidepanel-icon-button {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: deep

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)